src/views/ADempiere/ProcessActivity/processActivity.js

```diff
@@ -149,16 +149,28 @@ export default defineComponent({
           }
         }, () => {})
       } else if (activity.command === 'zoomIn') {
+        const parameters = root.isEmptyValue(activity.parametersList) ? activity.parameters : activity.parametersList
         zoomIn({
           uuid: activity.uuid,
           query: {
             ...root.$route.query,
-            ...activity.parametersList
+            ...parameters
           }
         })
+        setProcessParameters(activity.uuid, parameters)
       }
     }
 
+    function setProcessParameters(containerUuid, parameters) {
+      Object.keys(parameters).forEach(element => {
```

Comment:
please use the function `convertObjectToKeyValue`, to transform the object into the expected array, and change the mutation updateValueOfContainer to the mutation updateValuesOfContainer, to prevent unnecessary loops through the values.

https://github.com/adempiere/adempiere-vue/blob/develop/src/utils/ADempiere/valueFormat.js#L34

```js
const parametersList = convertObjectToKeyValue({
  object: parameters
})
root.$store.commit('updateValuesOfField', {
  containerUuid,
  attributes: parametersList
})
```